client/webserver/site/src/js/state.ts

```diff
@@ -1,12 +1,22 @@
-const darkModeCK = 'darkMode'
-const authCK = 'dexauth'
-const popupsCK = 'popups'
-const pwKeyCK = 'sessionkey'
-
 // State is a set of static methods for working with the user state. It has
 // utilities for setting and retrieving cookies and storing user configuration
 // to localStorage.
 export default class State {
+  // Cookie keys.
+  static DarkModeCK = 'darkMode'
+  static AuthCK = 'dexauth'
+  static PopupsCK = 'popups'
+  static PwKeyCK = 'sessionkey'
+  // Local storage keys (for data that we don't need at the server).
+  static LoggersLK = 'loggers'
+  static RecordersLK = 'recorders'
+  static LastMarketLK = 'selectedMarket'
+  static DepthZoomLK = 'depthZoom'
+  static LastMMMarketLK = 'mmMarket'
+  static OptionsExpansionLK = 'mmOptsExpand'
+  static LeftMarketDockLK = 'leftmarketdock'
+  static SelectedAssetLK = 'selectedasset'
```

Comment:
After @chappjc mentioned [those 2 local storage values I was unaware of](https://github.com/decred/dcrdex/pull/1975#discussion_r1051113367) I though it might be worth grouping all local storage / cookie entries for easier tracking, like this ^.

Which I kinda started doing already in this PR, and [now covered all the rest](https://github.com/decred/dcrdex/pull/1975/commits/5c25bcf46ef2426f5e6ed2d90cc99cec33e82c00).

Reply:
Whoops, forgot to add `static NotificationsLK = 'notifications'`, now done.